### ISSUE TITLE
display stays black

### ISSUE THREAD

**Issue by wjcarpenter** [airy10/esphome-m5stickC#7]
I tried using exactly the sample YAML file, verbatim except for:

- changing to a font file that I already had in my environment
- removing references to the axp192 (different problem :-( )

So far, I can get nothing to appear on the screen. I even simplified the display lambda down to just a constant string, but still nothing. Logging and blinky lights show that button presses are working as expected. I am using the latest esphome (1.14.5). 

Is it possible that something has changed in esphome in the last couple of months to throw a wrench into this?
My m5stick-c is recently purchased. Could there be a hardware change that causes the problem? (The demo program that came on the m5stick-c worked fine until I flashed over it.)

I'm just starting to troubleshoot this, so I'm wondering if someone has hit this already and knows the simple answer.

**Comment by wjcarpenter** [comment]
Do I need to be using the `dev` or `beta` branch of esphome, or is the released version OK?

**Comment by wjcarpenter** [comment]
NVM. 

It was the removing of the axp192 references that seems to have been behind the screen not waking up properly. The reason I removed that (and, in fact, all of the `sensor:` block) was because esphome was complaining about a YAML problem that I couldn't isolate except by doing that. 

Today, I started over with a fresh git clone of the repo, using an exact copy of the sample YAML. After I changed the three lines to use a font I already had locally, it worked without further problems. I'll probably never figure out what I did subtly wrong the first time around.

Sorry for any inconvenience if anyone was thinking about what the problem might have been!
